src/issuer.spec.ts

```diff
@@ -62,8 +62,32 @@ describe('Issuer', () => {
     };
 
     const sdVCClaimsDisclosureFrame: DisclosureFrame = { person: { _sd: ['name', 'age'] } };
+    const sdVCHeader = {
```

Comment:
Please format the `sdVCHeader` for better readability. Here's a suggested format:

```typescript
const sdVCHeader = {
  kid: '1b94c',
  x5c: [
    `MIIDQjCCAiqgAwIBAgIGATz/FuLiMA0GCSqGSIb3DQEBBQUAMGIxCzAJB
    gNVBAYTAlVTMQswCQYDVQQIEwJDTzEPMA0GA1UEBxMGRGVudmVyMRwwGgYD
    ...`
  ],
};

Reply:
I wanted a valid example (valid x.509) and the indentation would make this invalid unless something changed regarding multi-line strings in typescript in recent years.
I've just changed to an array of 2 1-line strings to check that x5c would work properly - that way the test does not look ugly but core functionality is tested.